docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,190 @@
+.. _metrics_guide:
+
+====================
+Metrics Guide
+====================
+
+In this guide we'll explain how to customize the metrics that deepchecks uses to validate and monitor your model.
+Controlling the metrics helps you shape the checks and suites according to the specifics of you use case.
+
+**Structure:**
+
+* `Default metrics <#default-metrics>`__
+* `Alternative metrics <#alternative-metrics>`__
+* `List of Supported Strings <#list-of-supported-strings>`__
+* `Custom Metrics <#custom-metrics>`__
+
+
+Default metrics
+===============
+All of the checks that evaluate model performance, such as SingleDatasetPerformance, come with
+default metrics.
+
+The default scorers by task type are:
+
+Tabular
+_______
+
+Classification, binary or multiclass averaged over the classes:
+
+*   Accuracy
+*   Precision
+*   Recall
+
+Multiclass classification per class:
+
+*   F1
+*   Precision
+*   Recall
+
+Regression:
+
+*   Negative RMSE (negative RMSE so that the convention that greater is better is kept)
+*   Negative MAE
+*   R2
+
+Vision
+______
+
+Classification:
+
+*   Precision
+*   Recall
+
+Object detection:
+
+*   Mean average precision
+*   Mean average recall
+
+To run a check with the default metrics, run it without passing any value to the "scorer" parameter. For example:
+
+:doc:`ClassPerformance </checks_gallery/vision/model_evaluation/plot_class_performance>`:
+
+.. code-block:: python
+
+    check = ClassPerformance()
+    result = check.run(train_ds, test_ds, mnist_model)
+
+
+Alternative metrics
+===================
+Sometimes the defaults don't fit the specifics of the use-case.
+If this is the case, you can pass a list of supported metric strings or a dict in the format {metric name: metric} as a
+parameter to the check.
+
+The metrics in the dict can be existing:
+
+*   `Ignite metrics <https://pytorch.org/ignite/metrics.html#complete-list-of-metrics>`__ for vision.
+    An Ignite Metric is a class with the methods: reset, compute, and update, that iterates over batches of data and
+    aggregates the result.
+*   `Sklearn scorers <https://scikit-learn.org/stable/modules/model_evaluation.html>`__ for both vision and tabular.
+    An Sklearn Scorer is a function that accepts the parameters: (model, x, y_true), and returns a score with the
+    convention that higher is better.
+*   strings from Deepchecks' `supported strings <#list-of-supported-strings>`__ for both vision and tabular.
+
+or `your own implementation <#custom-metrics>`__.
```

Comment:
Not 100% clear what is the distinction between passing your own sklearn scorer / ignite metric (like you show in the end of this file), and actually implementing your own code (not shown here, meaning implementing your own code with the appropriate api).

Reply:
@noamzbr WDYM by "not shown here"? I gave 2 code examples: 1 for tabular and 1 for vision

Reply:
Yes but you didn't give an example of a real custom metric - e.g. implementing your own function that has the signature (model, X, y) but is not built using make_scorer. Not saying we should necessarily do an example, but that this is a viable option should be explicitly mentioned.